src/zkevm_specs/evm_circuit/instruction.py

```diff
@@ -567,17 +567,17 @@ def sub_word(self, minuend: Word, subtrahend: Word) -> Tuple[Word, FQ]:
 
         return Word((diff_lo, diff_hi)), FQ(borrow_hi)
 
-    def mul_word_by_u64(self, multiplicand: Word, multiplier: Expression) -> Tuple[Word, FQ]:
+    def mul_word_by_u64(self, multiplicand: Word, multiplier: Expression) -> Word:
         multiplicand_lo, multiplicand_hi = multiplicand.to_lo_hi()
 
         quotient_lo, product_lo = divmod((multiplicand_lo * multiplier.expr()).n, 1 << 128)
         quotient_hi, product_hi = divmod(
             (multiplicand_hi * multiplier.expr() + quotient_lo).n, 1 << 128
         )
 
-        product_bytes = product_lo.to_bytes(16, "little") + product_hi.to_bytes(16, "little")
+        self.constrain_zero(FQ(quotient_hi))
 
-        return Word((FQ(product_lo), FQ(product_hi))), FQ(quotient_hi)
+        return Word(FQ(product_lo), FQ(product_hi))
```

Comment:
I think the test is failing because the function signature is incorrect here.
The first argument is a tuple.
After the argument is fixed, the tests can pass on my local side.
```suggestion
        return Word((FQ(product_lo), FQ(product_hi)))
```

Reply:
@ChihChengLiang You're correct! Thanks a lot.

@KimiWu123 Fixed!